Barotrauma/BarotraumaShared/SharedSource/GameSession/GameModes/CoOpMode.cs

```diff
@@ -1,12 +1,91 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Barotrauma
 {
-    class CoOpMode : MissionMode
+    internal class CoOpMode : MissionMode
     {
+        private float endRoundTimer;
+
         public CoOpMode(GameModePreset preset, IEnumerable<MissionPrefab> missionPrefabs) : base(preset, ValidateMissionPrefabs(missionPrefabs, MissionPrefab.CoOpMissionClasses)) { }
 
         public CoOpMode(GameModePreset preset, MissionType missionType, string seed) : base(preset, ValidateMissionType(missionType, MissionPrefab.CoOpMissionClasses), seed) { }
+
+#if CLIENT
```

Comment:
It'd be better to move the client-specific code to a partial class.

Reply:
Fixed.